### DISCUSSION TITLE
Issue with nnUNet use : "from nnunetv2.inference.predict_from_raw_data import predict_from_raw_data"

### DISCUSSION THREAD

**Discussion by P20L0** [Project-MONAI/MONAI#7378] [General]
It seems that there is an issue in the runner since _predict_from_raw_data_ is not defined properly.
I've tried several things, importing the predictor class but it doesn't work... 

I get this message 

> File "/home/iadi.lan/psou/Environnements/MONAI/monai/apps/nnunet/nnunetv2_runner.py", line 827, in predict
    from nnunetv2.inference.predict_from_raw_data import predict_from_raw_data
ImportError: cannot import name 'predict_from_raw_data' from 'nnunetv2.inference.predict_from_raw_data' (/home/iadi.lan/psou/Environnements/nnUNet/nnunetv2/inference/predict_from_raw_data.py)

Any ideas of what's happening ?
Thank you very much :).

**Comment by KumoLiu** [comment]
Hi @P20L0, it's been a known issue since the update from nnunet. Would suggest using v2.1 as a workaround.
https://github.com/Project-MONAI/MONAI/issues/7013
Thanks!

**Reply by P20L0** [reply]
Thank you very much :). Indeed, my previous issue has vanished !

Then I'm using v2.1 but it seems I have another small problem with my arguments apparently. I'm still using the caller :
`!python -m monai.apps.nnunet nnUNetV2Runner predict_ensemble_postprocessing --input_config "./input.yaml" \`

And I get : 

`TypeError: monai.apps.nnunet.nnunetv2_runner.nnUNetV2Runner.predict() got multiple values for keyword argument 'list_of_lists_or_source_folder'`

I don't really know what these 

> "multiple values"

 could be since I'm only using a string as input argument (as `--list_of_lists_or_source_folder '/home/p2o/data'`).

Many thanks !

**Reply by P20L0** [reply]
I think I've found the solution by commenting on the following lines !
This leaves the possibility of writing the arguments as _kwargs_.

```
 self.predict(
                    # list_of_lists_or_source_folder=source_dir,
                    # output_folder=output_dir,
                    # model_training_output_dir=model_folder,
                    use_folds=used_folds,
                    # save_probabilities=run_ensemble,
                    verbose=False,
                    overwrite=True,
                    **kwargs,
                )
```
